src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -272,7 +262,7 @@ object DepositHandler {
     }
 
     val fetchItems = getFetchTxt(bagDir).map(_.asScala).getOrElse(Seq())
-    val (fetchItemsInBagStore, itemsToResolve) = fetchItems.partition(bs.isBagStoreAware && _.getUrl.startsWith(bs.baseUrl))
+    val (fetchItemsInBagStore, itemsToResolve) = fetchItems.partition(bs.nonEmpty && _.getUrl.startsWith(bs.get.baseUrl))
```

Comment:
How would you use `map` or `getOrElse` here? I think that it just gets unnecessarily complicated.